apps/price_pusher/src/evm/evm.ts

```diff
@@ -187,8 +188,10 @@ export class EvmPricePusher implements IPricePusher {
     // are using this to remain compatible with the networks that doesn't
     // support this transaction type.
     let gasPrice =
-      Number(await this.customGasStation?.getCustomGasPrice()) ||
-      Number(await this.client.getGasPrice());
+      this.gasPrice !== undefined
+        ? this.gasPrice
+        : Number(await this.customGasStation?.getCustomGasPrice()) ||
```

Comment:
Using `||` to coalesce zeros is [generally considered bad practice](https://typescript-eslint.io/rules/prefer-nullish-coalescing) since it's surprising and unintuitive.

If I understand the intent of this code correctly, I think this would be better written as:

```tsx
let gasPrice = this.gasPrice ?? Number(await (this.customGasStation?.getCustomGasPrice() ?? this.client.getGasPrice()));
```